lib/console1984/sessions_logger/database.rb

```diff
@@ -33,7 +33,13 @@ def before_executing(statements)
     end
   end
 
-  def after_executing(statements)
```

Comment:
The method were already hanging there, so I took advantage by putting the fix there.